src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -139,6 +144,17 @@ public Server startServer() throws Exception {
         return server;
     }
 
+    List<AbstractIdleService> getBeforeHealthCheckServices(Injector injector) {
+        List<AbstractIdleService> beforeHealthCheckServices = new ArrayList<>();
+        InfluxdbReporterLifecycle influxdbReporterLifecycle = injector.getInstance(InfluxdbReporterLifecycle.class);
+        beforeHealthCheckServices.add(influxdbReporterLifecycle);
```

Comment:
How do you feel about instantiating + returning the list in one statement? Java 9 has `List.of(elements...)`, but for now we can use com.google.common's `return newArrayList(servicesAsArgumentList...)`.

Reply:
FWIW, in pre-9 I use Arrays.asList(...);

Reply:
Oh, sweet! Somehow I missed out on this for like a year and a half.

Reply:
Oh cool, TIL thanks!